### DISCUSSION TITLE
How to delete a wifi connection?

### DISCUSSION THREAD

**Discussion by gbr1** [iRobotEducation/create3_docs#109] [Questions] [answered]
Hi,
I would like to delete a saved wifi connection (the create3 always connects), how can I do?
It seems to be impossible, I also tried to leave blank the wifi forms and try to connect.

Thank you in advance.

**Comment by shamlian** [comment, marked as answer]
At the moment, the only way to do this is via a [factory reset](https://iroboteducation.github.io/create3_docs/webserver/about/#factory-reset).

**Reply by shamlian** [reply]
I've added an issue to our public tracker (#110) so we can fix in a future release.

**Reply by gbr1** [reply]
thx!
